### ISSUE TITLE
Markdown.Avalonia.Html nupkg has no dlls included

### ISSUE THREAD

**Issue by fifty-six** [whistyun/Markdown.Avalonia#127]
The 11.0.x packages seem to not have any dlls included - is this intended?
```
Markdown.Avalonia.Html.11.0.1
├── [Content_Types].xml
├── Markdown.Avalonia.Html.md
├── Markdown.Avalonia.Html.nuspec
├── package
│   └── services
│       └── metadata
│           └── core-properties
│               └── 0a9e1c629050405a93362db0b0b38344.psmdcp
└── _rels
```
I was trying to use the Html plugin for Markdown.Avalonia, but there doesn't seem to be a nuget package with it included.

**Comment by whistyun** [comment]
apologize for the late reply.

I released 11.0.2 packages.
These include assemblies.

**Comment by fifty-six** [comment]
Thanks!
